CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## Release 0.2.0
+
+This is a feature and bugfix release.
+
```

Comment:
Good call, I've broken the changes up into sections to match the previous release entry.